teams/4/page-1.html

```diff
@@ -23,6 +23,9 @@
   <link rel="stylesheet" href="../../css/navigation.css" />
   <link rel="stylesheet" href="../../css/main.css" />
   <link rel="stylesheet" href="../../css/normalize.css" />
+  <link rel="stylesheet" type="text/css" href="slick/slick.css" />
+  <link rel="stylesheet" type="text/css" href="slick/slick-theme.css" />
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css" />
```

Comment:
what is this? we are not using any external styles, remove

Reply:
fixed in the team-4 branch